cmake/umd_device.cmake

```diff
@@ -22,11 +22,21 @@ if($ENV{ENABLE_TRACY})
 endif()
 
 # MUST have the RPATH set, or else can't find the tracy lib
-set(LDFLAGS_ "-L${CMAKE_BINARY_DIR}/lib -Wl,-rpath,${CMAKE_BINARY_DIR}/lib ${CONFIG_LDFLAGS} -ldl -lz -lboost_thread -lboost_filesystem -lboost_system -lboost_regex -lpthread -latomic -lhwloc -lstdc++")
+set(LDFLAGS_ "-L${CMAKE_BINARY_DIR}/lib -Wl,-rpath,${CMAKE_BINARY_DIR}/lib ${CONFIG_LDFLAGS} -ldl -lz -lpthread -latomic -lhwloc -lstdc++")
 set(SHARED_LIB_FLAGS_ "-shared -fPIC")
 set(STATIC_LIB_FLAGS_ "-fPIC")
 
 set (CMAKE_CXX_FLAGS_ "--std=c++17 -fvisibility-inlines-hidden")
+foreach(lib ${BoostPackages})
+    set(CMAKE_CXX_FLAGS_ "${CMAKE_CXX_FLAGS_} -I${Boost${lib}_SOURCE_DIR}/include")
+endforeach()
+
+set(UMD_OUTPUT > /dev/null 2>&1)
```

Comment:
What's this for?

Reply:
To silence the UMD build output, with the new boost include paths, it's very chunky. It'll still output on CI or if someone sets `VERBOSE=1`